### ISSUE TITLE
Minor release (0.10.0)

### ISSUE THREAD

**Issue by bobmyhill** [geodynamics/burnman#332]
The most recent release of burnman is now *very* old. We should make another minor release. I've kicked the more major changes down the road (see Issue #388), but these things should be done before we release 0.10.0:
- [x] Merge #383 
- [x] Merge #384 
- [x] Merge #386
- [x] Merge #387
- [x] Merge #389 
- [x] Merge #394

**Comment by tjhei** [comment]
Should we use this issue to track the release? Can you link any PRs / issues that are still missing?

**Comment by tjhei** [comment]
What is the problem with the documentation?

**Comment by bobmyhill** [comment]
I've opened a new PR fixing the problem. sphinx changed some of the setup of conf.py.

**Comment by bobmyhill** [comment]
Apparently our readthedocs page has failed to update for two and a half years... now it's working again!!

**Comment by bobmyhill** [comment]
Hi @tjhei, I think that's all the changes we need to make for the release. Shall I do the rest?

**Comment by tjhei** [comment]
Do you know how to tag a release, create a GitHub release, release to pypi, etc.? I can do that today or tomorrow if you want.
The only useful thing to have would be a list of changes. I guess we are not keeping a changelog?

**Comment by bobmyhill** [comment]
I've just made a pre-release (for which I guess you've just received an email).
If you're happy with it, I can finalise it and release to pypi.

We haven't been keeping a changelog, but we should do that for 1.0.0.

**Comment by tjhei** [comment]
1. Can https://burnman.readthedocs.io/en/latest/ generate versions for specific tags on github? If yes, please enable it and link to that in the release.

2. You set the version incorrectly, I think. You need to set the correct version and tag that commit: https://github.com/geodynamics/burnman/blob/70fc0771bcbb4a5db521bfd8af2df037479004ce/burnman/version.py#L1

3. You need to bump the version on master (it is still incorrectly at 0.10.0-pre)

**Comment by tjhei** [comment]
Did you just tag master as v0.10.0 on github? I would not do it that way...

**Comment by tjhei** [comment]
> I've just made a pre-release

You mean you marked the release as a "prerelease" on github. This still means that you tagged v0.10.0 in git, see https://github.com/geodynamics/burnman/tree/v0.10.0

**Comment by bobmyhill** [comment]
Thanks for the advice! I think I've now set up the release correctly.
Uploading to pypi looks straightforward, but I'm not sure how it works if I jump in as a new maintainer?

**Comment by tjhei** [comment]
> Thanks for the advice! I think I've now set up the release correctly.

Yes. You will need to increase the master to 0.11.0-pre or 1.0.0-pre as well.



> Uploading to pypi looks straightforward, but I'm not sure how it works if I jump in as a new maintainer?

I don't know.

**Comment by bobmyhill** [comment]
Hi @thjei,

I got this message from `python -m twine upload --repository testpypi dist/*`:

> `The user 'bob.myhill' isn't allowed to upload to project 'burnman'. See https://test.pypi.org/help/#project-name for more information.`

So I think that either you can add me as a maintainer or the upload has to be done by you.

**Comment by tjhei** [comment]
Try again now.

**Comment by bobmyhill** [comment]
Brilliant, thanks @tjhei! All done :)
